go.mod

```diff
@@ -44,8 +45,9 @@ require (
 	github.com/jonboulle/clockwork v0.1.0 // indirect
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/looplab/fsm v0.1.0
-	github.com/onsi/ginkgo v1.8.0 // indirect
-	github.com/onsi/gomega v1.5.0 // indirect
+	github.com/mitchellh/go-homedir v1.1.0
```

Comment:
Let's replace this dependency to something else. This points to an individual maintained lib: https://github.com/mitchellh/go-homedir. Ideally, we do not want to depend on such lib.

Reply:
Done